Acd 1084 pos2 (#103)
Actindo-AG/Documentation: POS/Operation/02_SelectOffer.md

[!!Manage offers for POS](../Integration/07_ManageOffers.md)
[!!User interface Cashpoint](../UserInterface/01a_Cashpoint.md)

> # Record the purchase

Author: add offers to bill list

> The *Venduo POS* module offers several ways to select an offer and to search for it. Additionally, you can prepare further actions for editing the bill list, to ensure the protection of minors, to create returns, and to add product options.  

Reviewer: Record the purchase --> several ways to select an offer and to search for it -> Erster Satz passt nicht mehr zum Titel. Diese Info unter "Select an offer" weiter unten und hier Intro-Info anpassen?

Author: done

![Cashpoint](../../Assets/Screenshots/POS/Sales/Cashpoint/Cashpoint.png "[Cashpoint]")

1. Enable the ![Toggle](../../Assets/Icons/Toggle.png "[Toggle]")*Single scan* toggle in the bill list on the right side of the pay desk interface.   

    > [Info] The single scan works from the moment it is enabled. It does not apply backwards to offers that were already on the bill list at the time of activation.

    Even if the single scan function is active, you can edit the quantity of offers manually in the bill list, see [Edit the quantity of a product](./03_EditBillList.md#edit-the-quantity-of-a-product).

2. Select an offer, see [Select an offer](02_SelectOffer.md#select-an-offer).   
    The product is listed in the bill list.

3. Select the same offer again.   
    The same product is listed in an own entry in the bill list.

    ![Product single scan](../../Assets/Screenshots/POS/Sales/Cashpoint/ProductSingleScan.png "[Product single scan]")

## Create a return

Record a return, so that a negative quantity is posted in the bill list, and you can hand over the money to the customer. You can add multiple different offers as well as the required quantity of a single offer.   

> Note hat when you edit a return, all posts on the bill list must be returns. You cannot combine purchases and returns in one bill list. 

Reviewer: ```suggestion
Note that when you edit a return, all posts on the bill list must be returns. You cannot combine purchases and returns in one bill list. 
```

Author: done

#### Prerequisite

- Offers for POS are created, see [Manage offers for POS](../Integration/07_ManageOffers.md).
- Your company uses the *Allow negative quantities* setting. For detailed information, see [Allow negative quantities](../UserInterface/02a_GlobalSettings.md#allow-negative-quantities).
- There is no offer in the bill list yet.

#### Procedure

*Venduo POS > Sales > Select store and pay desk > Tab CASHPOINT*

![Cashpoint](../../Assets/Screenshots/POS/Sales/Cashpoint/Cashpoint.png "[Cashpoint]")

1. Enable the *Negative quantity* toggle.   
    The *Negative quantity* toggle is displayed in red.

    ![Negative quantity](../../Assets/Screenshots/POS/Sales/Cashpoint/BillList/BillListNegativeQuantity.png "[Negative quantity]")

2. Select the offer(s), see [Select an offer](#select-an-offer).   
    The offers are added to the bill list with a negative amount on the right side.   

>     ![Negative amount](../../Assets/Screenshots/POS/Sales/Cashpoint/BillList/BillListNegativeAmount.png "[Negative amount]")

Reviewer: Leerzeile fehlt

Author: done

    > [Info] The search already provides you with proposals for your number as you type.

3. Select the desired offer by clicking the corresponding offer in the list.   
    The offer is added to the bill list if no further features such as an [age verification](#verify-the-age) or a [product option](#add-a-product-option) are interposed. 

    > [Info] The number remains in the search bar when you add an offer to the bill list. When you click the search bar with that number again, the search results list for this number is displayed again.

### Select a pre-printed voucher offer 

> Select a voucher offer that allows you to identify and charge a pre-printed voucher. You can then scan the voucher code with a barcode scanner or add the card code manually. 

Reviewer: Select a voucher offer that allows you to identify and charge a pre-printed voucher. -> Damit verstehe ich nicht ganz, was ein pre-printed voucher ist.

Author: done

#### Prerequisites

> - Your company uses the KNISTR integration, with which vouchers can be pre-printed.

Author: done

#### Procedure

*Venduo POS > Sales > Select store and pay desk > Tab CASHPOINT* 

![Cashpoint](../../Assets/Screenshots/POS/Sales/Cashpoint/Cashpoint.png "[Cashpoint]")

1. Click the *Find offer by nam...* search bar in the upper left part of the tab.  
    The cursor is displayed in the search bar.

2. Enter the name or the SKU of the voucher you are searching for in the search bar.
    The offers that match the search are displayed in a list below the search bar.

    > [Info] The search already provides you with proposals as you type.

> 3. Select the desired voucher by clicking the corresponding offer in the list.   

Reviewer: Select the desired voucher offer by clicking the desired one in the list?

(Pre-printed) voucher offer -> pre-printed würde ich vielleicht weglassen (wenn, dann nur das erste Mal und dann erklären). Voucher offer im Sinne von "voucher" als Produkt zu kaufen wird manchmal nicht so deutlich. Eigentlich es ist einfach ein anderes Produkt, das man kaufen kann, und wo man noch Geldwert darauf aufladen kann, oder?

Author: man muss den Pre-printed von der normalen Voucher unterscheiden

    The *Voucher details* window is displayed.

    ![Voucher details](../../Assets/Screenshots/POS/Sales/Cashpoint/VoucherDetails.png "[Voucher details]")

4. Scan the voucher code with the barcode scanner or enter the code manually.     
    If the barcode was successfully scanned, the voucher code is automatically added to the *Voucher code* field. The cursor is placed in the *Amount* field, where a default value is provided, which you can overwrite.

5. Enter the amount in the *Amount* field. Note that this amount will be added to any amount already on the voucher.

6. Click the [SAVE] button.   
    The voucher with its newly created amount is added to the bill list.

<!--- NEU-->

> ## Verify the age

Reviewer: Vorschlag: Select an offer with age verification

Author: ne, das ist ja völlig verwirrend

Some offers are not suitable for children and young people, for example offers such as alcoholic beverages, cigarettes, or energy drinks. When you select an offer, Actindo checks whether an age verification is necessary before the offer is added to the bill list.

#### Prerequisites

- Offers for POS are created, see [Manage offers for POS](../Integration/07_ManageOffers.md).
- An age restriction for the offer is defined, for example 18 or 16.    
- The *Display age rating* setting is activated. For detailed information, see [Display age rating](../UserInterface/02a_GlobalSettings.md#display-age-rating).

#### Procedure

*Venduo POS > Sales > Select store and pay desk > Tab CASHPOINT* 

![Cashpoint](../../Assets/Screenshots/POS/Sales/Cashpoint/Cashpoint.png "[Cashpoint]")

1. Select an offer.   

>     If the offer requires an age verification, the *Age verification* window is displayed.

Reviewer: ```suggestion
    The *Age verification* window is displayed.
```
Wir haben oben gesagt, dass Age verification is eingestellt.

Author: s.o.

    ![Age verification](../../Assets/Screenshots/POS/Sales/Cashpoint/AgeVerification.png "[Age verification]")

2. Check the age of the customer. The *Age verification* window provides you with both the minimum age required and the date of birth that the customer must meet or fall below. Eventually, an explanatory text has been added.

    ![Age verification](../../Assets/Screenshots/POS/Sales/Cashpoint/AgeVerification.png "[Age verification]")
    

3. Click the [AGE OF BUYER CONFIRMED] button if the customer is allowed to buy this offer.  
    The offer is added to the bill list.

> 4. Click the [CANCEL] button if the customer is too young to buy this offer.   

Reviewer: Ist dieser Schritt wirklich nötig oder selbstverständlich?

Author: ja

    The offer is not added to the bill list.

## Add a product option

There are products where the buyer has the option of customizing them. If a store has the appropriate technical capabilities, this could be shirts with optional flocking, for example, or photos that a customer can have printed on mugs or other items. In these cases, a specific service is associated to an offer. This service is optional for customers and can be purchased additionally.

#### Prerequisites

- Offers for POS are created, see [Manage offers for POS](../Integration/07_ManageOffers.md).

> - Offers relevant to product options have been associated to the offers to which they are related, see [Create a product option](../Integration/07_ManageOffers.md#create-a-product-option).

Reviewer: **Offers** relevant to product options have been **associated to the offers** **to which they are related** -> verstehe nicht, was gemeint ist, sorry

Author: done

#### Procedure

*Venduo POS > Sales > Select store and pay desk > Tab CASHPOINT* 

![Cashpoint](../../Assets/Screenshots/POS/Sales/Cashpoint/Cashpoint.png "[Cashpoint]")

1. Select an offer.   
    If a product option is associated to the offer and your store offers this service, the *Choose options* window is displayed.

    ![Choose options](../../Assets/Screenshots/POS/Sales/Cashpoint/ChooseOption1.png "[Choose options]")

2. Present the various options to the customer.   
    + If the customer does not want to use the service, click the [SAVE] button.   
        The *Choose options* window is closed and the selected offer is added to the bill list.
    + If the customer wants to use the service, select the required service.  
        Depending on the service option, additional information might be required, for example, if a customer wants to have his/her name printed on a shirt. In this case, a field for adding this information is displayed.  

        ![Choose options](../../Assets/Screenshots/POS/Sales/Cashpoint/ChooseOption2.png "[Choose options]")
        

3. If applicable, enter the name or other information that might be required.

4. Click the [SAVE] button.  

>     The *Choose options* window is closed. The product option and the additionally edited information are added to the bill list.

Reviewer: ```suggestion
    The *Choose options* window is closed. The offer, the product option, and the additional(ly entered) information are added to the bill list.
```

Author: done
